src/cmd/build.js

```diff
@@ -12,9 +12,47 @@ import {createLogger} from '../util/logger';
 
 const log = createLogger(__filename);
 
+// Flow Types
+
+import type {OnSourceChangeFn} from '../watcher';
+import type {ExtensionManifest} from '../util/manifest';
+
+export type PackageCreatorFn =
+    (params: PackageCreatorParams) => Promise<ExtensionBuildResult>;
+
+export type ExtensionBuildResult = {
+  extensionPath: string,
+};
+
+export type PackageCreatorParams = {
+  manifestData?: ExtensionManifest,
+  sourceDir: string,
+  fileFilter: FileFilter,
+  artifactsDir: string,
+};
+
+export type BuildCmdParams = {
+  sourceDir: string,
+  artifactsDir: string,
+  asNeeded?: boolean,
+};
+
+export type BuildCmdExtraParams = {
+  manifestData?: ExtensionManifest,
+  fileFilter?: FileFilter,
```

Comment:
@kumar303 checked this again and it is working as expected, e.g.:

- if fileFilter is set to `null` (e.g. in test.build.js at line 85), flow raises a type error (because null in not compatible with the FileFilter type)
- if fileFilter is set to `undefined`, flow does not raise any type error (because the default value will be used, and it is a valid FileFilter for sure)